frontend/src/container/GridCardLayout/GridCard/WidgetGraphComponent.tsx

```diff
@@ -269,19 +269,21 @@ function WidgetGraphComponent({
 			</Modal>
 
 			<div className="drag-handle">
-				<WidgetHeader
-					parentHover={hovered}
-					title={widget?.title}
-					widget={widget}
-					onView={handleOnView}
-					onDelete={handleOnDelete}
-					onClone={onCloneHandler}
-					queryResponse={queryResponse}
-					errorMessage={errorMessage}
-					threshold={threshold}
-					headerMenuList={headerMenuList}
-					isWarning={isWarning}
-				/>
+				{headerMenuList.length > 0 && (
```

Comment:
headerList will be always present if i am not sure

Reply:
`headerMenuList={handle.active ? [] : widgetActions}`
Here we are setting it empty. Tested it. Working as expected.